runtime/compiler/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -4254,10 +4254,10 @@ J9::Z::TreeEvaluator::dwrtbarEvaluator(TR::Node *node, TR::CodeGenerator *cg)
    // For rdbar and wrtbar nodes we first evaluate the children we need to
    // handle the side effects. Then we delegate the evaluation of the remaining
    // children and the load/store operation to the appropriate load/store evaluator.
-   TR::Register *valueReg = cg->evaluate(node->getFirstChild());
    TR::Node *sideEffectNode = node->getSecondChild();
+   TR::Register *valueReg = cg->evaluate(node->getFirstChild());
    TR::Register *sideEffectRegister = cg->evaluate(sideEffectNode);
-
+   
```

Comment:
Can we remove the extra space added here?

Reply:
Don't worry about the extra spaces. I'm working on getting rid of them.

Reply:
Ah okay. I wasn't able to find any indication that you're already working on removing them. Hence why I left the comment.

Reply:
https://github.com/eclipse/openj9/pull/6234#issuecomment-504559868

Reply:
Okay, if in the future you can be more specific and mention to not to comment on spacing/formatting issues as you are addressing them then it'll be helpful. It can be hard to tell what exactly we can review since you did mention that the PR has been opened for review.

Or alternatively you can let us know when the PR is officially ready for review after you have made all your changes.